includes/derivatives.inc

```diff
@@ -151,6 +151,7 @@ function islandora_fits_add_datastream($object, $datastream_id, $file_uri) {
   try {
     $ingest = !isset($object[$datastream_id]);
     $mime_detector = new MimeDetect();
+    $techmd_dsid = variable_get('islandora_fits_techmd_dsid', 'TECHMD');
```

Comment:
Should `$techmd_dsid = variable_get('islandora_fits_techmd_dsid', 'TECHMD');` come before the `try`? If it stays here, will it be in scope within the accompanying `catch`?

Reply:
Wrote a little script to test this... it is in scope within the `try`.